series/static/series/search_series.css

```diff
@@ -0,0 +1,30 @@
+form {
```

Comment:
I dont know where this is defined, but I dont like the very large input fields for login/signup/profile...

Reply:
Same with update button on profile

Reply:
Fixed